database/migrations/mysql/1583505136_add_vsphere_pke_clusters_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `vsphere_pke_clusters` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `created_at` timestamp NULL DEFAULT NULL,
+  `updated_at` timestamp NULL DEFAULT NULL,
+  `deleted_at` timestamp NULL DEFAULT NULL,
+  `cluster_id` int(10) unsigned DEFAULT NULL,
+  `provider_data` json DEFAULT NULL,
```

Comment:
`provider_data` feels weird when this is a provider specific table. `spec` maybe?